operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -296,6 +296,34 @@ spec:
 #    ---
 #    view_only_mode: false
 
+##########
+#  ---
+#  extensions:
```

Comment:
You need to add these to the defaults/main.yml in the operator kiali-deploy role - add them here: https://github.com/kiali/kiali/blob/v1.12/operator/roles/kiali-deploy/defaults/main.yml#L72

And since you are adding a new top-level group, you have to add this group to the vars/main.yml here: https://github.com/kiali/kiali/blob/v1.12/operator/roles/kiali-deploy/vars/main.yml#L54